cirq/protocols/circuit_diagram_info_protocol.py

```diff
@@ -102,37 +102,45 @@ class CircuitDiagramInfoArgs:
         precision: The number of digits after the decimal to show for numbers in
             the text diagram. None means use full precision.
         qubit_map: The map from qubits to diagram positions.
+        include_tags: Whether to print tags from TaggedOperations
     """
 
     UNINFORMED_DEFAULT = None  # type: CircuitDiagramInfoArgs
 
-    def __init__(self, known_qubits: Optional[Iterable['cirq.Qid']],
-                 known_qubit_count: Optional[int], use_unicode_characters: bool,
+    def __init__(self,
+                 known_qubits: Optional[Iterable['cirq.Qid']],
+                 known_qubit_count: Optional[int],
+                 use_unicode_characters: bool,
                  precision: Optional[int],
-                 qubit_map: Optional[Dict['cirq.Qid', int]]) -> None:
+                 qubit_map: Optional[Dict['cirq.Qid', int]],
+                 include_tags: bool = True) -> None:
```

Comment:
Should probably add some tests for the new attribute (equality group test, etc.).

Reply:
Added some lines for equality testing, but couldn't think of much else to test here.